layers/landcover/style.json

```diff
@@ -51,20 +39,13 @@
         "visibility": "visible"
       },
       "paint": {
-        "fill-color": "#6a4",
-        "fill-opacity": {
-          "base": 0.1,
+        "fill-color": {
           "stops": [
-            [
-              7,
-              0
-            ],
-            [
-              9,
-              0.1
-            ]
+            [7, "rgba(218, 234, 214, 1)"],
+            [12, "rgba(199, 218, 199, 1)"]
           ]
         },
+        "fill-opacity": 0.5,
```

Comment:
This opacity will be overwritten by Kibana, right? Does that cause issues when Kibana layer opacity is set to 100%?

Maybe I'm not understanding the root issue correctly?

Reply:
The issue is only with dynamic opacity. With fixed opacity as we are setting here, Kibana will multiply this value by the opacity set for the whole map. That process is carried out here: 

https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/classes/layers/ems_vector_tile_layer/ems_vector_tile_layer.tsx#L413

My first attempt was to modify that `_setOpacityForType` method to process an opacity object by taking the `base` property but then I decided that it was a better solution to fix this directly on the styles.